pemFioi/conceptViewer-1.0-mobileFirst.js

```diff
@@ -442,7 +442,7 @@ function getConceptsFromBlocks(includeBlocks, allConcepts, context) {
       // times a documentation
       var includedConceptIds = [];
       // We remove all concepts which have no "python" attribute
-      var filteredConcepts = allConcepts.filter(concept => concept.python && concept.python != []);
+      var filteredConcepts = allConcepts.filter(function(concept) { return concept.python && concept.python != [] });
```

Comment:
Missing semicolon at the end of the return.

Reply:
Sorry, have added it in next commit